exporter/util/epoch/parse.go

```diff
@@ -11,8 +11,6 @@ import (
 
 const (
 	frontendSourceDateEpochArg = "build-arg:SOURCE_DATE_EPOCH"
-
-	KeySourceDateEpoch = "source-date-epoch"
```

Comment:
Not a big issue, but epoch is not technically image exporter specific and is used by all exporters.

Reply:
Ah, yes, I did think about it too, but still wanted to have it as an option of the `image` exporter.
I could leave this as a duplicate here, but we still would depend on `exptypes` here due to `exptypes.ExporterEpochKey`.

We could have a "top level" exporter keys that are common for all exporters, and then have something like:
```go
OptKeySourceDateEpoch ImageExporterOptKey = commonexptypes.OptKeySourceDateEpoch
```

Reply:
Pushed at a separate commit, let me know what you think.

Reply:
Move the `exptypes.ExporterEpochKey` also to that pkg. Then I think the "common" pkg is fine.

Reply:
Done.